charts/veritable-ui/templates/deployment.yaml

```diff
@@ -252,6 +252,8 @@ spec:
               value: {{ .Values.schemaPolicy | quote }}
             - name: ISSUANCE_CRED_DEF_POLICY
               value: {{ .Values.credentialPolicy | quote }}
+            - name: DEMO_MODE 
+              value: {{ include "veritable-ui.demoMode" . | quote }}
```

Comment:
Can't this just be `value: {{ .Values.demoMode | quote }}`?

Reply:
if not where is the template you're including?

Reply:
it meant to be, should have updated with https://github.com/digicatapult/helm-charts/pull/114/commits/560cb19456befbf31ddffbfd544b9b43c8ae079d commit